styles/mixins/tabs.js

```diff
@@ -0,0 +1,65 @@
+module.exports = function () {
+  return function (mixin, number, borderRadius) {
+    var styles = {
+      '.tabs': {
+        display: 'flex',
+        'flex-wrap': 'wrap',
+        'max-width': '100%',
+        position: 'relative',
+        'list-style': 'none',
+        'border-radius': borderRadius,
+      },
+      '.tab': {
+        display: 'none',
+        '&:first-of-type:not(:last-of-type) + label': {
+          'border-top-right-radius': '0',
+          'border-bottom-right-radius': '0'
+        },
+        '&:not(:first-of-type):not(:last-of-type) + label': {
+          'border-radius': '0'
+        },
+        '&:last-of-type:not(:first-of-type) + label': {
+          'border-top-left-radius': '0',
+          'border-bottom-left-radius': '0'
+        },
+        '&:checked + label': {
+          cursor: 'default'
+        },
+        '& + label': {
+          'box-sizing': 'border-box',
+          display: 'block',
+          'flex-grow': '3',
+          'border-radius': borderRadius + ' ' + borderRadius + ' 0 0',
+          'text-decoration': 'none',
+          cursor: 'pointer',
+          'user-select': 'none',
+        },
+        '&-content': {
+          position: 'absolute',
+          left: 0,
+          'z-index': '-1',
+          width: '100%',
+          'border-radius': borderRadius,
+          opacity: 0,
+          transform: 'translateY(-3px)'
```

Comment:
It does.  Take a look at the example.  We could remove it and see if there is issues with it as well.  Thoughts?